cmds/tui.c

```diff
@@ -2167,6 +2167,7 @@ static void tui_main_loop(struct opts *opts, struct ftrace_file_handle *handle)
 				switch ((long)cmd->data) {
 				case TUI_SESS_REPORT:
 					win = &report->win;
+					tui_window_move_home(win);
```

Comment:
You'd better doing it consistently - the same should go to 'graph' and 'info' as well.

Reply:
I applied the same thing to graph and info as well.